docs/modules/server.rst

```diff
@@ -21,6 +21,10 @@ For example, a client request could be {‘endpoint’: ‘get_guild_count’, 
 This JSON is processed upon a request being made, and checks for a registered route matching the name of the endpoint supplied.
 It then calls the method linked to said route and returns the payload to the client.
 
-.. automodule:: discord.ext.ipc.server
+.. currentmodule:: discord.ext.ipc.server
+
+.. py:function:: route
+
```

Comment:
Is [IpcServerResponse](https://github.com/Ext-Creators/discord-ext-ipc/blob/master/discord/ext/ipc/server.py#L32) appropriate for documentation?

Reply:
I'll ask Logan if its something used externally, I haven't used it yet myself so I didn't think it was public. Let me ask him and read the code then Ill respond